src/components/app.js

```diff
@@ -10,6 +10,13 @@ import Footer from './footer';
 let store = createStore();
 
 export default class App extends Component {
+	componentDidMount() {
+		if (typeof window !== undefined) {
+			// This triggers a second render :/
+			store.setState({ url: window.location.pathname });
```

Comment:
Need to find a way to initialize the store with the correct location instead of updating it later.

Reply:
can do it when initializing the store: https://github.com/preactjs/preact-www/pull/305/files#r302571204